.github/workflows/simple-pbt-e2e-test.yaml

```diff
@@ -0,0 +1,33 @@
+name: E2E Test with simple-pbt
+on:
+  - pull_request
+
+jobs:
+  e2e:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Test Env
+        uses: ./.github/workflows/template-setup-e2e-test
+        with:
+          kubernetes-version: ${{ matrix.kubernetes-version }}
+
+      - name: Run e2e test with ${{ matrix.experiments }} experiments
+        uses: ./.github/workflows/template-e2e-test
+        with:
+          experiments: ${{ matrix.experiments }}
+          # Comma Delimited
+          trial-images: simple-pbt
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # Detail: https://hub.docker.com/r/kindest/node
+        # TODO (tenzen-y): We need to consider running tests on more kubernetes versions.
+        # kubernetes-version: ["v1.20.15", "v1.21.12", "v1.22.9", "v1.23.6", "v1.24.1"]
+        kubernetes-version: ["v1.21.12", "v1.22.9", "v1.23.6"]
+        # Comma Delimited
+        experiments: ["simple-pbt"]
```

Comment:
We need to sync `experiments` name with the manifest name.
For example, If this experiments name is "simple-pbt", we need to modify [this](https://github.com/kubeflow/katib/blob/806354cfcf1cf169f00b575db67c867ed7dff3f9/examples/v1beta1/hp-tuning/pbt.yaml) manifest name from pbt.yaml to simple-pbt.yaml.